packages/backend/src/app.module.ts

```diff
@@ -15,6 +16,7 @@ import { UsersModule } from './users/users.module';
       isGlobal: true,
     }),
     UsersModule,
+    MongooseModule.forRoot('mongodb://localhost/nest'),
```

Comment:
can we make this part of the .env

Reply:
Sorry forgot something: can you please delete `src/users/users.service.ts` and the spec file. And change the controller to use the DB service instead

Reply:
I should remove the `dto` as well right?

Reply:
I think it is fine because the DTO will need annotations for swagger

Reply:
could you go through PR checklist as well please